json-csharp/IteratingOverJsonObject/IteratingOverJsonObject/JsonIteration.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Text.Json;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace IteratingOverJsonObject;
+
+public class JsonIteration
+{
+    public string Json { get; set; }
+    public string Name { get; set; }
+    public int Age { get; set; }
+    public string Department { get; set; }
+    public int ItemsCount { get; set; }
```

Comment:
As suggested in the article review, these variables don't add much value and they can be local variables to each method (and we can use that to print)